apps/betterangels-backend/accounts/schema.py

```diff
@@ -73,12 +73,13 @@ def generate_magic_link(self, info: Info, data: MagicLinkInput) -> MagicLinkResp
     @strawberry_django.mutation(extensions=[HasPerm(perms=[ClientProfilePermissions.ADD])])
     def create_client_profile(self, info: Info, data: CreateClientProfileInput) -> ClientProfileType:
         with transaction.atomic():
-            client_profile_data: dict = strawberry.asdict(data)
-            user_data = client_profile_data.pop("user") or {}
-
             user = get_current_user(info)
             permission_group = get_user_permission_group(user)
 
+            client_profile_data: dict = strawberry.asdict(data)
+            user_data = client_profile_data.pop("user") or {}
+            hmis_profiles_data = client_profile_data.pop("hmis_profiles", [])
```

Comment:
just a nitpick, but i expect data to be a `dict`, not a `list`
```suggestion
            hmis_profiles = client_profile_data.pop("hmis_profiles", [])
```